src/java_bytecode/generate_java_generic_type.cpp

```diff
@@ -19,13 +20,19 @@
 /// \param fqn_java_type The java type we want to pretty print.
 /// \return The pretty printed type if there was a match of the
 //  qualifiers, or the type as it was passed otherwise.
-static std::string pretty_print_java_type(const std::string &fqn_java_type)
+std::string pretty_print_java_type(const std::string &fqn_java_type)
```

Comment:
Given that this pretty print function doesn't appear to do anything in anyway specific to Java generics, should it perhaps be moved into `java_bytecode/java_utils.cpp/.h` ?

Reply:
This was just done.